doc/supportedPlatformVerions.md

```diff
@@ -238,16 +242,19 @@
 - 3.9.0b1
 - 3.9.0
 - 3.9.1rc1
+- 3.9.12
```

Comment:
As discussed separately, @qianz2 will send a separate PR to remove these last references to 3.9.12. We don't want to trigger a full rebuild.